ci/test_wheel_raft_dask.sh

```diff
@@ -11,7 +11,8 @@ RAPIDS_PY_WHEEL_NAME="raft_dask_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels
 RAPIDS_PY_WHEEL_NAME="pylibraft_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./local-pylibraft-dep
 python -m pip install --no-deps ./local-pylibraft-dep/pylibraft*.whl
 
-python -m pip install "raft_dask-${RAPIDS_PY_CUDA_SUFFIX}[test]>=0.0.0a0" --find-links dist/
+# echo to expand wildcard before adding `[extra]` requires for pip
+python -m pip install -v "$(echo ./dist/raft_dask_${RAPIDS_PY_CUDA_SUFFIX}*.whl)[test]"
```

Comment:
copied this commend and pattern from https://github.com/rapidsai/raft/blob/8ef71de26b01458f02f36ad96c1b3017cf985cc5/ci/test_wheel_pylibraft.sh#L10-L11